### PR TITLE
Bookstore: add reducers and actionsAdd reducers actions

### PR DESCRIPTION
**Pull request by RaphDasilva**
## In this Milestone, I;
- installed redux toolkit
- Created application files using the ducks pattern
- Added a slice of state for books and categories
- Added reducers to add a book, and remove a book
- Added a reducer to check the status in categoriesSlice
- Fixed eslint errors

### CODE REVIEW

File: src/redux/books/booksSlice.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  reducers: {
+    addBook: (state, action) => {
+      const bookId = action.payload;
+      state.books.concat(bookId);
+    },
+    removeBook: (state, action) => {
+      const bookId = action.payload;
+      state.books.filter((book) => book.item_id !== bookId);
+    },
+  },
```
Comment (CollinsTatang) [inline src/redux/books/booksSlice.js:19]: - [ ] Please the `addBook`  and `removeBook` actions are not working at the level of the User Interface try to fix that.

- **Adding and Removing actions must be implemented according to [project requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/bookstore/add_reducers_an_actions.md) .**

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @RaphDasilva , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/redux/books/booksSlice.js (hunk 1 above)
Comment (RaphDasilva) [reply src/redux/books/booksSlice.js:19]: Hi, I think this is not in the requirement

Comment (RaphDasilva) [reply src/redux/books/booksSlice.js:19]: prove

Comment (CollinsTatang) [reply src/redux/books/booksSlice.js:19]: Hi @RaphDasilva 

You are right I use the old project requirement sorry about that.
Please your dashboard will be updated soon.

**Review by CollinsTatang — APPROVED** [review]
# Approved 👏  
### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:
- No linter errors :heavy_check_mark: 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
### **Congratulations!** 🎉 
image



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**